llvm/unittests/Analysis/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 set(LLVM_LINK_COMPONENTS
   Analysis
   AsmParser
+  CodeGen
```

Comment:
Missing Codegen components caused linker issues when building with the flag `-DBUILD_SHARED_LIBS=ON`